FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Add plic.h and update README, docmuments

Add plic and update README, docmuments

Description
-----------
Add plic.h driver (platform-level-interrupt controller) : optional for app-developer (if they want to enable and use it)
Update the README / Documents

Test Steps
-----------
Locally tested all the tests using TestRunner.c


Related Issue
-----------
None

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
